### ISSUE TITLE
Crash after desynced

### ISSUE THREAD

**Issue by Thoralf-M** [iotaledger/goshimmer#598]
## GoShimmer version

0.2.0

## Errors

```
2020-06-30T23:02:15+02:00	INFO	Sync	sync/plugin.go:168	no message received in 300 seconds, marking node as desynced
2020-06-30T23:02:15+02:00	INFO	Sync	sync/plugin.go:191	monitoring for synchronization, awaiting 3 anchor point messages to become solid
2020-06-30T23:02:15+02:00	PANIC	Sync	sync/plugin.go:249	Failed to start as daemon: existing background worker is still running: Sync-Monitor is still running
github.com/iotaledger/goshimmer/plugins/sync.monitorForSynchronization
	/__w/goshimmer/goshimmer/plugins/sync/plugin.go:249
github.com/iotaledger/goshimmer/plugins/sync.markDesynced
	/__w/goshimmer/goshimmer/plugins/sync/plugin.go:113
github.com/iotaledger/goshimmer/plugins/sync.monitorForDesynchronization.func3
	/__w/goshimmer/goshimmer/plugins/sync/plugin.go:169
github.com/iotaledger/hive.go/daemon.(*OrderedDaemon).runBackgroundWorker.func1
	/github/home/go/pkg/mod/github.com/iotaledger/hive.go@v0.0.0-20200625105326-310ea88f1337/daemon/daemon.go:118
```

**Comment by Dr-Electron** [comment]
Got the same error with v0.2.1

**Comment by capossele** [comment]
Fixed by #670
